### PR TITLE
Allow for anonymous access to stateless endpoints

### PR DESCRIPTION
**Pull request by RonnyB71**
## Description
This allows for anonymous users to access and use stateless apps, or stateless forms in apps.

### CODE REVIEW

File: src/Altinn.App.Api/Models/LayoutSets.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        /// The layout set id.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The data type bound to the layout set
+        /// </summary>
+        public string DataType { get; set; }
+    }
+}
```
Comment (tjololo) [inline src/Altinn.App.Api/Models/LayoutSets.cs:32]: Nitpicking again, add a blank line at the end of the file

File: src/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs [hunk 2]
```diff
@@ -56,6 +55,44 @@ public async Task StatelessData_Get_ObjectSucessfullyPrefilledAndCalculated()
...
+            Mocks.Apps.Ttd.AnonymousStateless.Models.Veileder dataObject = JsonConvert.DeserializeObject<Mocks.Apps.Ttd.AnonymousStateless.Models.Veileder>(responseContent);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, res.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("nonexisting")]
+        [InlineData("")]
+        public async Task GetAnonymous_CorrectDataType_ShouldReturnBadRequest(string dataType)
```
Comment (tjololo) [inline src/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs:80]: Copy/Paste error? Testname name suggest correct dataType should return bad request 
```suggestion
        public async Task GetAnonymous_BadDataType_ShouldReturnBadRequest(string dataType)
```

**Review by tjololo — COMMENTED** [review]
Looks good, some suggestions/nitpick 🙈

File: src/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs (hunk 2 above)
Comment (RonnyB71) [reply src/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs:80]: Thx, copy/paste error:)

**Review by tjololo — APPROVED** [review]
:godmode:

**Pull request merged**